### ISSUE TITLE
After upgrading to version 5.0.4~ynh1 from version 5.0.3~ynh1 the 2FAuth page is blank with a bunch of errors in dev console

### ISSUE THREAD

**Issue by CodeShakingSheep** [YunoHost-Apps/2FAuth_ynh#60]
### Describe the bug

After upgrading to version 5.0.4~ynh1 from version 5.0.3~ynh1 the 2FAuth page is blank (see screenshot). Also, there are a bunch of errors in the dev console.

![image](https://github.com/YunoHost-Apps/2FAuth_ynh/assets/19874562/9a8a803b-d38d-462b-a1ad-96d5953583c2)

### Context

- Hardware: *VPS bought online*
- YunoHost version: 11.2.10.1
- I have access to my server: *Through SSH | through the webadmin*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: *no*
- If upgrading, current package version: *from 5.0.3~ynh1 to 5.0.4~ynh1*

### Steps to reproduce

I just hit the updgrade button in webadmin.

### Expected behavior

2FAuth version 5.0.4~ynh1 should display the page correctly.

### Logs

```
Access to manifest at 'https://<YNH_DOMAIN>.com/yunohost/sso/?r=aHR0cHM6Ly9hdXRoLnVuc2Vlbi1ob3Jpem9ucy5jb20vbWFuaWZlc3QuanNvbg==' (redirected from 'https://<2FAUTH_DOMAIN>.com/manifest.json') from origin 'https://<2FAUTH_DOMAIN>.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
<YNH_DOMAIN>.com/yunohost/sso/ 
        
        
       Failed to load resource: net::ERR_FAILED
2app-06176200.js:5 TypeError: Cannot read properties of null (reading 'nextSibling')
    at nextSibling (app-06176200.js:1:53657)
    at Ct (app-06176200.js:1:45292)
    at Bt (app-06176200.js:1:45052)
    at Re (app-06176200.js:1:44890)
    at Gr (app-06176200.js:1:45406)
    at Re (app-06176200.js:1:44680)
    at v (app-06176200.js:1:37336)
    at j (app-06176200.js:1:45811)
    at mount (app-06176200.js:1:32548)
    at Ym.t.mount (app-06176200.js:1:64591)
ue @ app-06176200.js:5
<2FAUTH_DOMAIN>.com/:1 Access to internal resource at 'https://<YNH_DOMAIN>.com/yunohost/sso/' (redirected from 'https://<2FAUTH_DOMAIN>.com/manifest.json') from origin 'https://<2FAUTH_DOMAIN>' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by mat-mo** [comment]
Same here.

**Comment by jarod5001** [comment]
I am having the same problem but with a new install while setting the app only available for admins (since it's a one user app and I'm the only admin).

**Comment by Thovi98** [comment]
Could someone please try to upgrade by `sudo yunohost app upgrade 2fauth -u https://github.com/Thovi98/2FAuth_ynh/tree/testing --debug` and see if the issue persists?

**Comment by jarod5001** [comment]
> Could someone please try to upgrade by `sudo yunohost app upgrade 2fauth -u https://github.com/Thovi98/2FAuth_ynh/tree/testing --debug` and see if the issue persists?

It's working 👍

**Comment by CodeShakingSheep** [comment]
Thanks a lot for the PR @Thovi98. Apparently package version 5.1.0 has been pushed to master branch, but your fix hasn't been pushed to master yet. So, as I updated this app normally on my server I was still running into this issue. I will wait for your PR to be merged and will report back then.

**Comment by CodeShakingSheep** [comment]
With the merged PR the upgrade to version 5.1.1 worked. So, I'm closing this.
